itou/templates/siaes/includes/_configure_jobs_list.html

```diff
@@ -1,5 +1,6 @@
 {% load bootstrap4 %}
 <table class="js-jobs-table table table-bordered table-striped table-responsive-sm text-center{% if not job_descriptions %} d-none{% endif %}">
+    <caption>Liste de métiers(s) exercé(s) dans votre structure</caption>
```

Comment:
```suggestion
    <caption>Liste des métiers exercés dans votre structure</caption>
```

Même s'il n'y a qu'un métier, une liste est censée en afficher plusieurs. Bon, à toi de juger. :)

Reply:
hi hi oui :)
c'est bien que tu repasses la dessus, les captions de tables et bientot les label d'input seront plus précis s'ils sont revus par les devs qui connaissent exactement de quoi il retourne. Moi je mets ce qui me semble bon mais sans plus de profondeur.